### ISSUE TITLE
I cannot reproduce the best result of Dss v2 

### ISSUE THREAD

**Issue by Sucran** [AceCoooool/DSS-pytorch#8]
I just copy your model, loss, optimizer definition, other settings keep the same except the lr was set as 1e-4 and epoch was set as 100. When epoch reaches 100, the training curve seems to converge, but I test the model, it produces MAE as 0.069, MaxFb as 0.880. Do you change any default setting in your training?

**Comment by AceCoooool** [comment]
I just use 1e-6 as learning rate. 
1. after 100 epoch, the loss change slow. (You can adjust your learning rate, maybe 1e-4 is too large. For example, use [dynamic learning rate](https://pytorch.org/docs/master/optim.html#torch.optim.lr_scheduler.ReduceLROnPlateau))
2. You can use log(loss) to amplify you loss curve to see whether it is converged（it's unfriendly in visdom）

**Comment by tuhinkm** [comment]
I cant reproduce the best results from the paper. Has anyone done it ? Kindly share how...
